pkg/controller/servicebindingrequest/reconciler.go

```diff
@@ -116,7 +116,7 @@ func (r *Reconciler) Reconcile(request reconcile.Request) (reconcile.Result, err
 		return RequeueOnNotFound(err)
 	}
 
-	retriever := NewRetriever(ctx, r.client, plan)
+	retriever := NewRetriever(ctx, r.client, plan, instance.Spec.MountPathPrefix)
```

Comment:
@baijum Shouldn't this be `EnvVarPrefix` instead of `MountPathPrefix`?
considering
`MountPathPrefix is the prefix for volume mount` and `EnvVarPrefix is the prefix for environment variables`

or should it be both?

Reply:
Yes

Reply:
Done. 01d0ba5ecdb225b20d6686d20af74bb02d5eac8d